Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM node:14 AS development
+
+ENV USERNAME timetracker
+ENV HOME /home/${USERNAME}
+
+RUN useradd -ms /bin/bash ${USERNAME}
+
+WORKDIR ${HOME}/time-tracker-ui
+COPY [^.env]* . .
+RUN chown ${USERNAME}:${USERNAME} -R ${HOME}
```

Comment:
Why is this needed? the home folder already belongs to the new user "timetracker"

Reply:
Yes home folder belongs to the new user, but when I copy files inside of that folder this belongs to the root instead of the new user, for that reason I have some permission issues when I try to execute the project in the container and also when I try to install npm packages, that's why I need do that. Currently I change this to give permissions to the work directory for be more specific, because home actually belongs to the new user.